lib/widgets/banner/banner.dart

```diff
@@ -0,0 +1,148 @@
+import 'package:flex_ui/flex_ui.dart';
+import 'package:flex_ui/tokens/sizes.dart';
+import 'package:flutter/material.dart';
+import 'package:widgetbook/widgetbook.dart';
+import 'package:widgetbook_annotation/widgetbook_annotation.dart' as widgetbook;
+
+enum FlexBannerImageLayout {
+  /// Image is firstly visible, the rest of the element are shown after the
+  /// image (i.e. on the right if the orientation is horizontal, at the bottom
+  /// if orientation is vertical)
+  first,
```

Comment:
Minor, but probably **start** and **end** vs. first / last.

Reply:
Of course! 🤦  Will fix.